sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -133,6 +129,27 @@ class StatusTracking(Base):
     container_command = Column(String)
     processing_time = Column(Integer)
 
+    __table_args__ = (
+        # Partial unique index to ensure only one INPROGRESS or COMPLETED for a record
+        # We do want to allow multiple FAILED records
+        # NOTE: This does not work with sqllite (testing) DBs, only postgres
+        Index(
+            "idx_unique_status",
```

Comment:
Just curious why you had this name different than others. Is it used somewhere?

Reply:
There is already a primary key index `id = Column(Integer, Identity(start=1, increment=1), primary_key=True)` on the database. This is a new (second) index which I've called `idx_unique_status`, so a new name for this index specifically and I'm indicating it is an index with `idx`.

Reply:
Ah ok. Thank you for clarifying that. Makes sense now!